sale_timesheet_line_exclude/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Sales Timesheet: exclude Timesheet Line from Sale Order",
-    "version": "14.0.1.0.0",
+    "version": "15.0.1.0.0",
     "category": "Sales",
     "website": "https://github.com/OCA/timesheet",
     "author": "CorporateHub, " "Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "CorporateHub, Odoo Community Association (OCA)",
```

That line could be cleaned up a bit.